src/ecs/input/bundle.rs

```diff
@@ -0,0 +1,48 @@
+//! ECS input bundle
+
+use app::ApplicationBuilder;
+use config::Config;
+use ecs::ECSBundle;
+use ecs::input::{Bindings, InputHandler};
+use error::Result;
+
+/// Bundle for adding the InputHandler and input bindings
+///
+/// # Errors
```

Comment:
Should be `## Errors`

Reply:
I just followed what @csherratt has done with the ApplicationBuilder etc.